src/templates/Brochure/BrochureTemplate.tsx

```diff
@@ -0,0 +1,300 @@
+            <a className="text-sm text-zinc-500 break-all" href={footnote} target="_blank">
+            <sup>1</sup>
+            {footnote}
+            </a>
+            </>
+          )}
+        </div>
+        <div className="basis-3/4">{children}</div>
+      </div>
+      <div className="grow"></div>
+      {footer}
+      <div className="text-sm absolute bottom-0 right-10 p-2 text-gray-800">{page}</div>
+    </div>
+  </div>
+);
+
+export const ParagraphsWithLinks: React.FC<{ bodies?: string[], 
+  links?: { title?: string, url: string}[],
+  linkClassNames: string
+  startWithLink?: boolean }> = ({ bodies, links, linkClassNames, startWithLink = false }) => {
+    if (!bodies || !links) return <></>
+    if (startWithLink) { 
+      return (
+      <>
+      {links.map((item, i) => (
+        <>
+          <a className={linkClassNames} href={item.url} target="_blank">{item.title ? item.title : item.url}</a>
+          {bodies[i] && <span>{bodies[i]}</span>}
+        </>
+      ))}
+      </>)
+    } else {
+      return (
+        <>
+        {bodies.map((item, i) => (
+          <>
+            <span>{item}</span>
+            {links[i] && <a className={linkClassNames} href={links[i].url} target="_blank">{links[i].title ? links[i].title : links[i].url}</a>}
+          </>
+        ))}
+        </>)
+    }
+}
+
+const Page1: React.FC<{ document: BrochureDocument }> = ({ document }) => {
+  const contents = document.page1.contents;
+  return (
+    <BrochurePage
+      description={document.shared.tradeTrust.description}
+      backgroundStyle={{ backgroundImage: `url(${govtechCurve})`, backgroundPosition: "-58% 120%", backgroundSize: "45%" }}
+      page={1}
+      logo={ttLogo}
+      footnote={document.page1.footnote}
+      header={document.page1.header}
+    >
+      
+      {contents.map((section, i) => {
+        if (i === contents.length - 1)
+          return (
+            <>
+              <h5>{section.subheader}</h5>
+              {section.bodyAsList?.[0]}
+              <b>{section.bolded?.[0]}</b>
+              {section.bodyAsList?.[1]}
+              <b>{section.bolded?.[1]}</b>
+              {section.bodyAsList?.[2]}
+            </>
+          );
+        else {
+          return (
+            <>
+              <h5>{section.subheader}</h5>
+              <div>{section.body}</div>
+              <br />
+            </>
+          );
+        }
+      })}
+    </BrochurePage>
+  );
+};
+
+const Page2: React.FC<{ document: BrochureDocument }> = ({ document }) => {
+  const contents = document.page2.contents;
+  return (
+    <BrochurePage
+      description={document.shared.openAttestation.description}
+      backgroundStyle={{ backgroundImage: `url(${govtechCurve})`, backgroundPosition: "110% -70%", backgroundSize: "50%", transform: "scale(-1)" }}
+      page={2}
+      logo={oaLogo}
+    >
+      <h5>{contents[0].subheader}</h5>
+      <div>{contents[0].body}</div>
+      <br />
+      <div>{contents[1].body}</div>
```

Comment:
there's abit of cognitive load on the various `[num]`, say what number renders what data.

the structure i see for each page is:
1. header = `BrochureHeader`
2. heading = title of page.
3. aside = left panel.
4. body = right panel, this could be raw html to keep things direct, rather with all the conditionals now
5. footer = optional, it happens only on last page in this case

would this be better? or keep the complicated part static, don't have to conform to schema so we can lose the conditionals

Reply:
oh my intention was to keep it as reusable as possible, the content is different for each page so I decided to opt for the `contents` field and handle each one within each `Page` component